templates/SilverStripe/SelectUpload/SelectUploadField.ss

```diff
@@ -4,20 +4,20 @@
             <div class="ss-uploadfield-item-info">
                 <label class="ss-uploadfield-item-name">
                     <% if $CanSelectFolder %>
-                        <small><% _t('UploadField.ADDTO', 'Add to') %>
+                        <small><%t SilverStripe\\SelectUploadField.ADDTO 'Add to' %>
```

Comment:
Can this please match the class name?
`SilverStripe\\SelectUpload\\SelectUploadField.ADDTO`?

Same for all other 4 :)

Reply:
done :)